static_src/actions/user_actions.js

```diff
@@ -146,7 +148,7 @@ const userActions = {
     });
   },
 
-  addUserRoles(roles, apiKey, userGuid, entityGuid, entityType) {
+  addUserRoles(resource_role_name, user_role, userGuid, entityGuid, entityType) {
```

Comment:
ahh sorry for the confusing conversation.
we were saying `apiKey` -> `resource`
and you can revert `resource_role_name` to `roles` and leave `roles` as-is for now.
we will address the naming then.